rebuild-amplify/get_failed_amplify_builds.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# README: 
+# Update the branch_name and specified_date_time to filter the failed builds from that date/time
+
+# Disable the pager for AWS CLI output
+export AWS_PAGER=""
+
+branch_name="staging-lite"
```

Comment:
should we pass this as cli option so people can choose thier desired branch?

Reply:
@seaerchin no bw to do this now, preferring to merge this first for future use. Can do as enhancement if required